tests/test_text_generation_example.py

```diff
@@ -89,41 +97,51 @@
             ("gpt2-xl", 1, False, 51.61471298016438),
         ],
     }
+    MODEL_OUTPUTS = {
```

Comment:
Can this be also used for fp8 test?

Reply:
i have not tried but perhaps it can be a later effort

Reply:
Yeah I think we should add it for fp8 output check as well - with fp8, and int4. Maybe for another PR.